web/Services/LandlordService.cs

```diff
@@ -79,25 +81,66 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
         return ILandlordService.LandlordRegistrationResult.Success;
     }
 
+    public async Task<(ILandlordService.LandlordRegistrationResult result, int id)> RegisterLandlord(

```

Comment:
I'd be tempted to return the whole `LandlordDbModel` on success, rather than just the id. It's slightly clearer what's going on that way.

I think also change the method above to have the same return type, for consistencies sake.